integration/configs.go

```diff
@@ -22,16 +22,21 @@ const (
 	cortexConfigFile       = "config.yaml"
 	cortexSchemaConfigFile = "schema.yaml"
 	blocksStorageEngine    = "tsdb"
+	clientCertFile         = "certs/client.crt"
+	clientKeyFile          = "certs/client.key"
+	rootCertFile           = "certs/root.crt"
```

Comment:
I would call it `caCertFile` to keep the naming consistent with the Cortex config.